packages/cna-template/template/nuxt/package.js

```diff
@@ -0,0 +1,75 @@
+module.exports = {
+  apply (pkg, generator) {
+    // edge
+    const { cliOptions = {} } = generator.sao.opts
+    const edge = cliOptions.edge ? '-edge' : ''
+    if (edge) {
+      delete pkg.dependencies.nuxt
+      pkg.dependencies['nuxt-edge'] = 'latest'
+    }
+
+    const { scripts } = pkg
+    const { features, language = [], linter = [], runtime = [] } = generator.answers
+
+    // Linter
+    const eslint = linter.includes('eslint')
+    const lintStaged = eslint && linter.includes('lintStaged')
+    const stylelint = linter.includes('stylelint')
+    const prettier = linter.includes('prettier')
+
+    if (!eslint) {
+      delete pkg.scripts.lint
+      delete pkg.devDependencies['@nuxtjs/eslint-config']
+      delete pkg.devDependencies['@nuxtjs/eslint-module']
+      delete pkg.devDependencies['babel-eslint']
+      delete pkg.devDependencies.eslint
+      delete pkg.devDependencies['eslint-plugin-nuxt']
+    }
+    if (!lintStaged) {
+      delete pkg.husky
+      delete pkg['lint-staged']
+      delete pkg.devDependencies.husky
+      delete pkg.devDependencies['lint-staged']
+    }
+    if (!stylelint) {
+      lintStaged && delete pkg['lint-staged']['*.{css,vue']
```

Comment:
Is this missing the trailing `}`?

Reply:
@clarkdo

Reply:
Thanks for the pointing.

Reply:
Will be fixed in https://github.com/nuxt/create-nuxt-app/pull/493/